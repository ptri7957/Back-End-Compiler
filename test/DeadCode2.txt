( (test (n)
    (0 (ld r1 x)
       (lc r2 y)
       (ld r3 x)
       (ret r3) ) ) )