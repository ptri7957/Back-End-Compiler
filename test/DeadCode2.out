( (test (n)
    (0 (ld r3 x)
       (ret r3) ) ) )